frontend/src/app/main/data/workspace.cljs

```diff
@@ -1842,6 +1842,26 @@
       (assoc-in state [:workspace-local :inspect-expanded] expanded?))))
 
 
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; File Library persistent settings
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+
+(defn set-file-library-listing-thumbs?
+  [listing-thumbs?]
+  (ptk/reify ::set-file-library-listing-thumbs?
+    ptk/UpdateEvent
+    (update [_ state]
+      (assoc-in state [:workspace-global :file-library-listing-thumbs?] listing-thumbs?))))
```

Comment:
I think the function name and the state keyword should not end with `?`
Especially when the function is not a predicate.

Reply:
Fixed